doc/source/serve/http-guide.md

```diff
@@ -1,12 +1,37 @@
-# Calling Deployments via HTTP
+# HTTP with Serve
 
 
+
+This section should help you:
+
+- Understand how to configure HTTP server location
+- Understand how to send HTTP request to serve deployment
+- Understand how Ray Serve integrate with FastAPI
+- Understand how to use customized HTTP Adapter
+
+
+## Configuring HTTP Server Locations
+
+By default, Ray Serve starts a single HTTP server on the head node of the Ray cluster.
+You can configure this behavior using the `http_options={"location": ...}` flag
+in {mod}`serve.start <ray.serve.start>`:
+
+- "HeadOnly": start one HTTP server on the head node. Serve
+  assumes the head node is the node you executed serve.start
+  on. This is the default.
+- "EveryNode": start one HTTP server per node.
+- "NoServer" or `None`: disable HTTP server.
+
+:::{note}
+Using the "EveryNode" option, you can point a cloud load balancer to the
+instance group of Ray cluster to achieve high availability of Serve's HTTP
+proxies.
+:::
+
 (serve-http)=
 
 ## Calling Deployments via HTTP
 
-### Basic Example
-
 When you create a deployment, it is exposed over HTTP by default at `/{deployment_name}`. You can change the route by specifying the `route_prefix` argument to the {mod}`@serve.deployment <ray.serve.api.deployment>` decorator.
 
 ```python
```

Comment:
Put this in active voice with the subject.

When you create a deployment, Serve exposes your deployment over HTTP by default at ...

Any request to the Serve HTTP server at /counter is routed to the deployment’s __call__ method with a [Starlette Request object](https://www.starlette.io/requests/) as its sole argument.